File: CmdMents/TimesAndSeasons/EatUnleavenedBreadDuringPassover.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CmdMents.TimesAndSeasons
{
    class EatUnleavenedBreadDuringPassover : ObserveNewYearInFirstMonth
    {
        public EatUnleavenedBreadDuringPassover()
        {
            base.AlternateText = null;
            base.Book = CommandmentBook.Exodus;
            base.CanBeCarriedOutOnlyInIsrael = false;
            base.CanBeCarriedOutToday = true;
            base.Chapter = 12;
            base.CommandmentType = CmdMents.CommandmentType.Positive;
            base.FollowedByChristians = CommandmentObedience.None;
            base.FollowedByMessianics = CommandmentObedience.Obeyed;
            base.FollowedByObservantJews = CommandmentObedience.Obeyed;
            base.Number = 114;
            base.ShortSummary = "Eat unleavened bread during Passover and Unleavened Bread.";
            base.Text = "On the first month you are to eat bread made without yeast, from the evening of the fourteenth day until the evening of the twenty-first day.";
            base.Verse = 18;
        }
    }
}

File: EtzMitzvot/Commandments/TimesAndSeasons/eatUnleavenedBreadDuringPassover.ts
///<reference path='../commandmentBase.ts' />

class EatUnleavenedBreadDuringPassover extends CommandmentBase {
    constructor() {
        super();

        this.shortSummary = "Eat unleavened bread during <br />Passover and Unleavened Bread.";
        this.text = "On the first month you are to eat bread made without yeast, from the evening of the fourteenth day until the evening of the twenty-first day.";
        this.book = TorahBook.Exodus;
        this.chapter = 12;
        this.verse = 18;

        this.canBeCarriedOutToday = true;
        this.requiresLivingInIsrael = false;
        this.requiresTemple = false;
        this.commentary = null;

        this.type = CommandmentType.Positive;
        this.christianObservance = CommandmentObedience.None;
        this.messianicObservance = CommandmentObedience.Binding;
        this.jewishObservance = CommandmentObedience.Binding;
        this.number = 114;

        this.children = [

        ];
    }
}  